usrsctplib/netinet/sctputil.c

```diff
@@ -2009,7 +2009,7 @@ sctp_timeout_handler(void *t)
 			 type, inp, stcb, net));
 		SCTP_STAT_INCR(sctps_timosecret);
 		(void)SCTP_GETTIME_TIMEVAL(&tv);
-		inp->sctp_ep.time_of_secret_change = tv.tv_sec;
+		inp->sctp_ep.time_of_secret_change = (unsigned int)tv.tv_sec;
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/f2c2b601285fec3289c483088ffef255598b585f.